content/glance.pl.md

```diff
@@ -0,0 +1,109 @@
+
+Normatywne specyfikacje techniczne są na stronach [www.w3.org/TR/WCAG22](https://www.w3.org/TR/WCAG22/) (oryginalna wersja angielska) i [https://www.w3.org/Translations/WCAG21-pl/](https://www.w3.org/Translations/WCAG21-pl/) (autoryzowane tłumaczenie WCAG 2.1 na język polski)
+
+## Wytyczne
+
+{::nomarkdown}
+{% include box.html type="start" title="Postrzegalne" h=3 class="large" %}
+{:/}
+
+-   Zapewnij **[alternatywy tekstowe](https://www.w3.org/WAI/WCAG22/quickref/#text-equiv)** dla treści nietekstowych.
+-   Zapewnij [**napisy rozszerzone i inne alternatywy**](https://www.w3.org/WAI/WCAG22/quickref/#media-equiv) dla multimediów.
+-   Twórz treści, które można **[prezentować na różne sposoby](https://www.w3.org/WAI/WCAG22/quickref/#content-structure-separation)**, w tym za pomocą technologii wspomagających, bez utraty znaczenia.
+-   Ułatwiaj użytkownikom  **[przeglądanie i słuchanie treści](https://www.w3.org/WAI/WCAG22/quickref/#visual-audio-contrast)**.
+
+{::nomarkdown}
+{% include box.html type="end" %}
+{:/}
+
+
+{::nomarkdown}
+{% include box.html type="start" title="Funkcjonalne" h=3 class="large" %}
+{:/}
+
+-   Zapewnij dostępność wszystkich funkcjonalności za pomocą **[klawiatury](https://www.w3.org/WAI/WCAG22/quickref/#keyboard-operation)**.
+-   Daj użytkownikom  **[wystarczająco czasu](https://www.w3.org/WAI/WCAG22/quickref/#time-limits)**  na przeczytanie i skorzystanie z treści.
+-   Nie używaj treści, które powodują **[drgawki](https://www.w3.org/WAI/WCAG22/quickref/#seizures-and-physical-reactions)** lub dolegliwości fizyczne.
+-   Pomagaj użytkownikom **[nawigować i znajdować treści](https://www.w3.org/WAI/WCAG22/quickref/#navigation-mechanisms)**.
+-   Ułatwiaj **[wprowadzanie danych inaczej niż klawiaturą](https://www.w3.org/WAI/WCAG22/quickref/#input-modalities)**.
+
+{::nomarkdown}
+{% include box.html type="end" %}
+{:/}
+
+{::nomarkdown}
+{% include box.html type="start" title="Zrozumiałe" h=3 class="large" %}
+{:/}
+
+-   Dbaj, aby tekst był **[czytelny i zrozumiały](https://www.w3.org/WAI/WCAG22/quickref/#meaning)**.
+-   Spraw, aby treść pojawiała się i działała w sposób **[przewidywalny](https://www.w3.org/WAI/WCAG22/quickref/#consistent-behavior)**.
+-   Pomagaj użytkownikom **[unikać błędów i je korygować](https://www.w3.org/WAI/WCAG22/quickref/#minimize-error)**.
+
+{::nomarkdown}
+{% include box.html type="end" %}
+{:/}
+
+{::nomarkdown}
+{% include box.html type="start" title="Solidne" h=3 class="large" %}
+{:/}
+
+-   Zapewnij jak największą **[kompatybilność](https://www.w3.org/WAI/WCAG22/quickref/#ensure-compat)** z aktualnymi i przyszłymi narzędziami użytkownika.
+
+{::nomarkdown}
+{% include box.html type="end" %}
+{:/}
+
+## Wersje
+
+Ta strona podsumowuje WCAG na poziomie wytycznych. Wytyczne są takie same w WCAG 2.1 i WCAG 2.2. Aby dowiedzieć się więcej o nowych kryteriach sukcesu, zobacz [[Nowe w WCAG 2.2]](/standards-guidelines/wcag/new-in-22/pl).
```

Comment:
It is not necessary (and not recommended) to add pl at the end of internal URLs: the website is designed to automatically link to a translation if one exists.